ims/exception/file_system_exceptions.py

```diff
@@ -41,6 +41,21 @@ def __str__(self):
         return self.name + " is Busy (Could have Clones)"
 
 
+class SnapshotBusyException(ImageBusyException):
+    """
+    This exception will be raised if user removes a protected snapshot
+    """
+    @property
+    def status_code(self):
+        ImageBusyException.status_code(self)
```

Comment:
@xukan missed this last time. I think you can remove this status_code here since super class's status code will be used.